connpool: make `call()` execution faster (#10438)
tarantool/tarantool: changelogs/unreleased/gh-10596-connpool-filter-support-alive-filtering.md

## feature/connpool

- Now `connpool.filter()` supports filtering only alive instances by

>   specifying `'alive'` mode (gh-10496).

Reviewer: The issue number seems wrong.

Author: Thank you. Fixed.

> - Now `connpool.filter()` supports filtering only alive instances by
>   specifying `'alive'` mode (gh-10596).

Reviewer 2: ```suggestion
- Now `connpool.filter()` supports a new filtering mode, `'alive'`, for filtering only 
  alive instances (gh-10596).
```

Author: Thank you. I've had to change the whole message due to the PR update. Please, review it one more time.
